travis-build.sh

```diff
@@ -15,7 +15,7 @@ then
     psql -d scale -U postgres -c "create extension postgis_topology;"
 
     export COVERAGE_FILE=$root/.coverage
-    coverage run --source='.' manage.py test
+    coverage run --source='.' manage.py test -v 2
```

Comment:
do we still need the verbosity cranked up?

Reply:
Turned off verbosity. Meant to do this yesterday and forgot. Thanks!